tests/backends/oracle/tests.py

```diff
@@ -83,3 +83,24 @@ def test_hidden_no_data_found_exception(self):
         finally:
             with connection.cursor() as cursor:
                 cursor.execute('DROP TRIGGER "TRG_NO_DATA_FOUND"')
+
+
+@unittest.skipUnless(connection.vendor == 'oracle', 'Oracle tests')
+class PasswordWithAtSignTest(TransactionTestCase):
```

Comment:
Can you use a more generic class name so it could be used for other tests?  (may be fine to rename and reuse HiddenNoDataFoundExceptionTest?)